File: app-java/src/test/java/tech/thdev/java_udemy_sample/class_01/_04_ConstructorInitTest.java
package tech.thdev.java_udemy_sample.class_01;

import org.junit.Test;

/**
 * Created by tae-hwan on 09/10/2016.
 * <p>
 * 클래스 사용하기
 */

public class _04_ConstructorInitTest {

    @Test
    public void test() throws Exception {
        _03_ConstructorInit.InnerClass init = new _03_ConstructorInit.InnerClass();
        System.out.println(init.getName());
    }
}

File: app-kotlin/src/test/java/tech/thdev/kotlin_udemy_sample/class_01/_04_ConstructorInitTest.kt
package tech.thdev.kotlin_udemy_sample.class_01

import org.junit.Test

/**
 * Created by tae-hwan on 09/10/2016.
 */
class _04_ConstructorInitTest {

    @Test
    @Throws(Exception::class)
    fun test() {
        println(_03_ConstructorInit.InitClass().name)
        println(_03_ConstructorInit.InitClass("ABC").name)
        println(_03_ConstructorInit.ConstructorInit("ConstructorInit...").name)
    }
}